authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
+
+from .models import Article
+
+
+class ArticleSerializer(serializers.ModelSerializer):
+    slug = serializers.CharField(
+        max_length=100,
+        validators=[UniqueValidator(
+            queryset=Article.objects.all(),
+            message="Title already exists, please enter a different Title")],
```

Comment:
I think this error message is confusing given that you say title already exists when referring to the slug.

Reply:
The title is what the user enters. Not the slug! We create the slug from the title